docs/imperial.md

```diff
@@ -2,15 +2,17 @@
 
 All nf-core pipelines have been successfully configured for use on the CX1 cluster at Imperial College London HPC.
 
-To use, run the pipeline with `-profile imperial`. This will download and launch the [`imperial.config`](../conf/imperial.config) which has been pre-configured with a setup suitable for the CX1 cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+To use, run the pipeline with `-profile imperial,standard`. This will download and launch the [`imperial.config`](../conf/imperial.config) which has been pre-configured with a setup suitable for the CX1 cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
```

Comment:
why do you need `-profile imperial,standard`? is it not working with just `-profile imperial`?

Reply:
I see, that's why you have the imperial profile in the config file.
I understand now.